src/main/scala/com/microsoft/hyperspace/actions/RefreshAction.scala

```diff
@@ -40,6 +40,20 @@ class RefreshAction(
     write(spark, df, indexConfig)
   }
 
+  /**
+   * Validate index is in active state for refreshing and there are some changes in
+   * source data file(s).
+   */
+  final override def validate(): Unit = {
+    super.validate()
+
+    if (currentFiles.equals(previousIndexLogEntry.allSourceFileInfos)) {
+      // scalastyle:off
+      println("here")
```

Comment:
I guess you had added below these two lines for quick debugging and you may wanna delete them now

// scalastyle:off
   println("here")